examples/baselines/decision-transformer/Readme.md

```diff
@@ -0,0 +1,21 @@
+# Decision Transformer
+
+This baseline is adapted from the `gym` example provided [here](https://github.com/kzl/decision-transformer/tree/master). This folder provides both a jupyter notebook version that can be run on Google Colab as well as a python script that can be used for training a model. A few environments have already been trained on and these runs can be viewed [here](https://wandb.ai/ag115115/decision-transformer).
+
```

Comment:
last thing, add instructions for how to install requirements (it should be obvious but we need to make peoples lives easier). Make sure to mention that they need to install maniskill2 first.

Reply:
I added this in my most recent commit.